components/button/style/index.less

```diff
@@ -142,9 +142,7 @@
   }
 
   &-background-ghost {
-    background: transparent!important;
```

Comment:
@iharpeshka Cause we must felllow design spec, if you want chang backgroud color & text color, you can setup styles by yourself.